spec/support/auth_helper.rb

```diff
@@ -1,6 +1,27 @@
 # frozen_string_literal: true
 
 module AuthHelpers
+  def sign_in_ui(user = nil)
+    user ||= FactoryBot.create(:user)
+    visit new_user_session_path
```

Comment:
Let's add an extra new line before it
```suggestion

    visit new_user_session_path
```

Reply:
Added in 67b5ce8